inc/Action/Resendpwd.php

```diff
@@ -97,7 +97,7 @@ protected function resendpwd()
 
             $user = io_readfile($tfile);
             $userinfo = $auth->getUserData($user, $requireGroups = false);
-            if (!$userinfo['mail']) {
+            if (!empty($userinfo['mail'])) {
```

Comment:
This check seems backwards now. It triggers an error message when the email is NOT empty.